presto-hive-hadoop2/bin/run_hive_s3_tests.sh

```diff
@@ -23,6 +23,10 @@ exec_in_hadoop_master_container /usr/bin/hive -e "CREATE EXTERNAL TABLE presto_t
 
 stop_unnecessary_hadoop_services
 
+# restart hive-metastore to apply S3 changes in core-site.xml
```

Comment:
Would it be possible to apply the s3-related changes before starting the container?

Reply:
Then the change would be much more invasive.